libs/openchallenges/ui/src/lib/footer/footer.component.html

```diff
@@ -6,13 +6,14 @@
         <li><a href="/about">About</a></li>
         <li><a href="/team">Meet Our Team</a></li>
         <li><a href="#">API</a></li>
+        <li><a href="https://github.com/Sage-Bionetworks/sage-monorepo/issues/new/choose">Report a Bug</a></li>
```

Comment:
Open the door to feature requests; less emphasis on bugs.

```suggestion
        <li><a href="https://github.com/Sage-Bionetworks/sage-monorepo/issues/new/choose">Submit a Feature or Bug Request</a></li>
```